File: help/pt_pt/exercicios_da_ficha_4/Ficha04_014/Ficha04_014.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

public class Ficha04_014 {

    //Main Function
    public static void main( String[] args) {
        System.out.print("Tabela de verdade do OU\n" );
        boolean var = true || true ;
        System.out.print("\n" + true + "\t||\t" + true + "\t= " + var );
        var = true || false ;
        System.out.print("\n" + true + "\t||\t" + false + "\t\t= " + var );
        var = false || true ;
        System.out.print("\n" + false + "\t\t||\t" + true + "\t= " + var );
        System.out.print("\n" + false + "\t\t||\t" + false + "\t\t= " + ( false || false ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:32

File: help/pt_pt/exercicios_da_ficha_4/Ficha04_014/Ficha04_014.c
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Tabela de verdade do OU\n");
    int var = 1 || 1 ;
    printf("\n%d\t||\t%d\t= %d",1 ,1 ,var );
    var = 1 || 0 ;
    printf("\n%d\t||\t%d\t\t= %d",1 ,0 ,var );
    var = 0 || 1 ;
    printf("\n%d\t\t||\t%d\t= %d",0 ,1 ,var );
    printf("\n%d\t\t||\t%d\t\t= %d",0 ,0 ,( 0 || 0 ) );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:32
